### ISSUE TITLE
Missing alt text

### ISSUE THREAD

**Issue by discospiff** [subuhifu/Lindner-Airlines#7]
alt text should describe the image, for someone who can't see the image.  What colors are used?  What is in the image?  How big is the image?

https://github.com/subuhifu/Lindner-Airlines/blob/e69d786c3c343c491415de731b5e86e8e260c3cb/Pages/About%20Us.cshtml#L28
